README.md

```diff
@@ -25,6 +25,27 @@ c-example`.
 - macOS
 - Windows
 
+## Add linenoize to a project
+
+Add linenoize as a dependency to your `build.zig.zon` file:
+```zig
+.dependencies = .{
+    .linenoize = .{
+        .url = "git+https://github.com/joachimschmidt557/linenoize.git#752446d1cafad2bdd58768baa7337a3e9a7d1bf2",
+        .hash = "linenoize-0.1.0-J7HK8OnXAACP-w7_hZWvBIy7xU7ccbeeh0ZfBbZIFgND",
+    },
+},
+```
```

Comment:
I think using the `zig fetch --save` command is more suitable here. So something along the lines of

```bash
zig fetch --save git+https://github.com/joachimschmidt557/linenoize.git#v0.1.0
```

Reply:
Thanks, you're right, updated